Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Projection/Feature/SubtreeTagging.php

```diff
@@ -173,44 +180,51 @@ private function moveSubtreeTags(
             SET h.subtreetags = (
               SELECT
                 JSON_MERGE_PATCH(
-                    IFNULL(JSON_OBJECTAGG(htk.k, null), \'{}\'),
-                    JSON_MERGE_PATCH(\'{}\', h.subtreetags)
+                    IFNULL(JSON_OBJECTAGG(htk.k, null), '{}'),
+                    JSON_MERGE_PATCH('{}', h.subtreetags)
                 )
               FROM
-                JSON_TABLE(r.subtreeTagsToInherit, \'$[*]\' COLUMNS (k VARCHAR(36) PATH \'$\')) htk
+                JSON_TABLE(r.subtreeTagsToInherit, '$[*]' COLUMNS (k VARCHAR(36) PATH '$')) htk
```

Comment:
```suggestion
                JSON_TABLE(r.subtreeTagsToInherit, '$[*]' COLUMNS (k VARCHAR(36) PATH '\$')) htk
```
dont we need to escape doller signs as we are in heredoc vs nowdoc?

Reply:
I don't think so, but I'll have to double-check. In any case – escaping it would be safer probably. But we should do it in all places
```suggestion
                JSON_TABLE(r.subtreeTagsToInherit, '\$[*]' COLUMNS (k VARCHAR(36) PATH '\$')) htk
```

Reply:
a i think because php cannot substitute anything here because it not something like `$someVariable` its treated as literal. Im fine either way as long as it works ^^ ;)